mifosng-provider/src/main/java/org/mifosplatform/infrastructure/configuration/domain/ConfigurationDomainService.java

```diff
@@ -49,4 +49,5 @@
 
     int getRoundingMode();
 
+    boolean isBackdatePenalties();
```

Comment:
please give a proper name for this method (like allowTransactionsOnNonWorkingDayEnabled) and  DB column name